### ISSUE TITLE
Docker ♥️

### ISSUE THREAD

**Issue by dasrecht** [notwaldorf/tiny-care-terminal#81]
Hi there

As I didn't get it to run natively on my (maybe broken) system I took a detour via Docker.

The documentation is very very rough but maybe it helps someone else: https://github.com/dasrecht/tiny-care-terminal-docker

Thank you and have a great day :)

**Comment by hackjutsu** [comment]
@dasrecht If tiny-care-terminal is running inside a docker, can it still get my git information on my host?

**Comment by dasrecht** [comment]
Yes, just mount your git repositories into the container. everything under /code is taken into account

```
    -v ~/code_directory:/code \ # Mount your code into the container
```

**Comment by hackjutsu** [comment]
@dasrecht 

Thanks for clarification:) Do we have to login the Docker to see the output?

**Comment by gauravchl** [comment]
Can we publish it to docker hub as well?
Would be awesome to pull image directly.
`docker pull tiny-care-terminal`

**Comment by notwaldorf** [comment]
@gauravchl I uhhhh don't know what a docker or a docker hub is 😳. If you tell me how, I can (or: if you want to do it, go for it!)

**Comment by dasrecht** [comment]
@gauravchl It's on docker hub already. Named it this way because i didn't want to conflict it with the other repository.

```
docker pull dasrecht/tiny-care-terminal-docker
```

As it's just a small thing people can just pull it from my account if needed. But feel free to jump in :)

@notwaldorf 👋 no worries it's just a different way to run the tiny-care-terminal

**Comment by gauravchl** [comment]
@notwaldorf  @dasrecht Cool! May be should keep Dockerfile here in same repo?! And add some guide in Readme?

@notwaldorf In short Docker = Virtualization + Containerization + Some Versioning 

- **Virtualization**: We can run tiny-care-terminal in windows/osx machine as a linux environment.

- **Containerization**: No need to install dependencies(git-standup, gitlog, imagemagick…). Just pull the `tiny-care-terminal` image from docker hub and run it. Image comes with it’s own host environment(Ubuntu/ Debian..) and other dependencies(git-standlaone, node,..) bundled in an image(internally layer of images actually). 

- **Versioning**: Create image, push to repo, pull from repo, create another image of same and add different tag, push again, pull specific image by tags, and run different versions.



Quick Start:

1. Install docker: https://docs.docker.com/docker-for-mac/install/#download-docker-for-mac
2. Pull tiny-care-terminal `docker pull dasrecht/tiny-care-terminal-docker `
3. Finally run it `docker run -it dasrecht/tiny-care-terminal-docker`


Some cool things you can do with docker:  https://www.youtube.com/watch?v=1qlLUf7KtAw&t=415s
Like running TOR Browser :) 
Official get started: https://docs.docker.com/get-started/

**Comment by mojoaxel** [comment]
@dasrecht Maybe you want to add a small section to the Readme on how to use tct with docker!?
